spec/spec_helper.rb

```diff
@@ -37,6 +37,7 @@
 end
 
 RSpec.configure do |c|
+  c.hiera_config = 'hiera.yaml'
```

Comment:
How do we deal with this in modulesync?

Reply:
![dealwithit](https://user-images.githubusercontent.com/341866/69013438-01453c80-0980-11ea-9606-0ae82e365eea.gif)